### PR TITLE
CVE-2008-2302

### PR DESCRIPTION
**Pull request by peteyb37**
CVE-2008-2302.yml filled out by ptb3609

### CODE REVIEW

File: cves/CVE-2008-2302.yml [hunk 1]
```diff
@@ -9,30 +9,35 @@ yaml_instructions: |
...
-curated: false
+curated: true
+
 reported_instructions: |
   What date was the vulnerability reported to the security team? Look at the
   security bulletins and bug reports. It is not necessarily the same day that the
   CVE was created.  Leave blank if no date is given.
   Please enter your date in YYYY-MM-DD format.
-reported_date:
+reported_date: 2008-05-14
```
Comment (dwelcheck) [inline cves/CVE-2008-2302.yml:26]: Where did you find a reported date?  The date you list appears to be the _announcement date_ from the Django site: https://www.djangoproject.com/weblog/2008/may/14/security/

File: cves/CVE-2008-2302.yml [hunk 2]
```diff
@@ -9,30 +9,35 @@ yaml_instructions: |
...
+
 announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE date. A good
   source for this is Chrome's Stable Release Channel
   (https://chromereleases.googleblog.com/).
   Please enter your date in YYYY-MM-DD format.
-announced_date:
+announced_date: 2008-05-23
```
Comment (dwelcheck) [inline cves/CVE-2008-2302.yml:34]: See above comment

File: cves/CVE-2008-2302.yml [hunk 3]
```diff
@@ -49,7 +54,15 @@ description_instructions: |
...
   security
-description:
+description: |
+  When logging into an administrator account, an attacker could perform XXS or 
+  cross site scripting to inject an action into a form.  This would target a user
+  and execute the code that was injected into the form.  This would give the
+  attacker access to the target's information and potentially access tokens.  
+  This could also allow the attacker to take actions on the site as the target.
+  In Django, the request path was not being escaped, which created an opportunity
+  for attackers to inject an XXS attack.
```
Comment (dwelcheck) [inline cves/CVE-2008-2302.yml:64]: typo: XSS attack

File: cves/CVE-2008-2302.yml [hunk 4]
```diff
@@ -61,14 +74,15 @@ bounty:
 reviews: []
 bugs: []
 repo:
+
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
```
Comment (dwelcheck) [inline cves/CVE-2008-2302.yml:81]: It appears there's a couple more fix commits in backport branches: https://www.djangoproject.com/weblog/2008/may/14/security/

File: cves/CVE-2008-2302.yml [hunk 5]
```diff
@@ -61,14 +74,15 @@ bounty:
...
-- commit: 50ce7fb57d79e8940ccf6e2781f2f01df029b5c5
-  note:
+- commit: 41635d2176f7a950498b020f335232ad9f734279
+  note: Changeset 7521, found on djangoproject.com
 - commit: 7791e5c050cebf86d868c5dab7092185b125fdc9
-  note:
+  note: Changeset 7527, found on djangoproject.com
 vccs:
```
Comment (dwelcheck) [inline cves/CVE-2008-2302.yml:86]: Did you look into these VCCs to see if they were applicable?  Also, can you write a description/note yourself to describe how the commits contribute to the vulnerability?

File: cves/CVE-2008-2302.yml [hunk 6]
```diff
@@ -120,10 +137,14 @@ discovered:
...
     If there is no evidence as to how this vulnerability was found, then please explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    There was no evidence as to how this was found.  I looked through securityfocus.com, securitytracker.com,
+    djangoproject.com and others and could not find discussion on who found it.  It was likely just someone who
+    stumbled across it as it was an easy fix.
+  automated: nil
```
Comment (dwelcheck) [inline cves/CVE-2008-2302.yml:144]: Please put a boolean here.

File: cves/CVE-2008-2302.yml [hunk 7]
```diff
@@ -120,10 +137,14 @@ discovered:
...
+  contest: false
+  developer: nil
```
Comment (dwelcheck) [inline cves/CVE-2008-2302.yml:146]: Please put a boolean here.

File: cves/CVE-2008-2302.yml [hunk 8]
```diff
@@ -138,8 +159,9 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: This is an XXS attack vulnerability
```
Comment (dwelcheck) [inline cves/CVE-2008-2302.yml:162]: typo: XSS

Also, can you provide any other rationale beyond the type of vulnerability?  Does the nature of it being a exploitable via a web form make it easier, etc.

File: cves/CVE-2008-2302.yml [hunk 9]
```diff
@@ -171,8 +196,9 @@ subsystem:
     In the name field, a subsystem name (or an array of names)
 
     e.g. clipboard, model, view, controller, mod_dav, ui, authentication
-  answer:
-  name:
+  answer: django/contrib/admin/views/decorators.py
```
Comment (dwelcheck) [inline cves/CVE-2008-2302.yml:199]: This isn't really a subsystem name, it's the file path.  Try to identify the _subsystem_/modules associated (ex: admin, views, etc.)

File: cves/CVE-2008-2302.yml [hunk 10]
```diff
@@ -194,8 +221,11 @@ i18n:
 
     Answer should be boolean. Write a note about how you came to the conclusions
     you did.
-  answer:
-  note:
+  answer: false
+  note: |
+    While internationalization can cause xxs vulnerabilities, this was simply a mistake
```
Comment (dwelcheck) [inline cves/CVE-2008-2302.yml:226]: typo: XSS

File: cves/CVE-2008-2302.yml [hunk 11]
```diff
@@ -279,10 +319,11 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
+CWE: CWE-70: Improper Neutralization of Input During Web Page Generation
+CWE_note: XXS
```
Comment (dwelcheck) [inline cves/CVE-2008-2302.yml:323]: XSS

File: cves/CVE-2008-2302.yml [hunk 12]
```diff
@@ -90,6 +105,7 @@ upvotes_instructions: |
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
 upvotes:
+
```
Comment (dwelcheck) [inline cves/CVE-2008-2302.yml:108]: +3 upvotes

**Comment by dwelcheck** [comment]
Your build is failing.  Go into the [build log](https://travis-ci.com/github/VulnerabilityHistoryProject/django-vulnerabilities/builds/158918159) and see if you can figure out where the errors are happening.

File: cves/CVE-2008-2302.yml [hunk 13]
```diff
@@ -181,10 +207,11 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
+  - commit: 41635d2176f7a950498b020f335232ad9f734279
+    note: When comparing to a different commit, it seems as if this one just reverts it.
```
Comment (dwelcheck) [inline cves/CVE-2008-2302.yml:211]: What is `it` here?  The bug fix?

File: cves/CVE-2008-2302.yml [hunk 14]
```diff
@@ -279,10 +319,11 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
+CWE: CWE-70: Improper Neutralization of Input During Web Page Generation
```
Comment (mesternefeld) [inline cves/CVE-2008-2302.yml:322]: This should just be 70, it'll likely fix the yaml

**Pull request merged**